src/charm.py

```diff
@@ -141,6 +149,26 @@ def __init__(self, *args):
 
         self.log_forwarder = LogForwarder(self)
 
+        self.loki_device_alert_rules_path = "./loki_alert_rules"
```

Comment:
This path (will) contains the charms' alert rules. I suggest you create a `device` sub-folder and point to that instead, as it is the case for grafana dashboards.